tests/resources/test_data_source_dir/test_data_source_data.py

```diff
@@ -1287,11 +1186,14 @@ def create_connection_to_sqlite3_db(db_path):
     return sqlite3.connect(db_path)
 
 
-def oracledb_create_connection():
-    return FakeConnection(oracledb_all_type_data, oracledb_all_type_schema, "oracledb")
-
+def create_connection_oracledb():
+    import oracledb
```

Comment:
nit: I remember we discussed moving dbapi live test to the dir: https://github.com/snowflakedb/snowpark-python/tree/main/tests/integ/datasource

this PR seems to be a good chance to do this minor test refactor.

Reply:
will do